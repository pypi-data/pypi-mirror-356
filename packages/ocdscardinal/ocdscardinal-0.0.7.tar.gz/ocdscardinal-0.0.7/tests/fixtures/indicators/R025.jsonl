{"ocid":"0","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"0"}]}]},"awards":[{"status":"active","suppliers":[{"id":"0"}]}]}
{"ocid":"1","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"1"}]}]},"awards":[{"status":"active","suppliers":[{"id":"1"}]}]}
{"ocid":"2","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"2"}]}]},"awards":[{"status":"active","suppliers":[{"id":"2"}]}]}
{"ocid":"3","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"3"}]}]},"awards":[{"status":"active","suppliers":[{"id":"3"}]}]}
{"ocid":"4","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"N"}]}]},"awards":[{"status":"active","suppliers":[{"id":"F"}]}]}
{"ocid":"5","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"5"}]}]},"awards":[{"status":"active","suppliers":[{"id":"5"}]}]}
{"ocid":"6","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"6"}]}]},"awards":[{"status":"active","suppliers":[{"id":"6"}]}]}
{"ocid":"7","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"7"}]}]},"awards":[{"status":"active","suppliers":[{"id":"7"}]}]}
{"ocid":"8","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"8"}]}]},"awards":[{"status":"active","suppliers":[{"id":"8"}]}]}
{"ocid":"F","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"F"}]}]},"awards":[{"status":"active","suppliers":[{"id":"F"}]}]}
{"ocid":"cancelled tender","bids":{"details":[{"status":"valid","tenderers":[{"id":"F"}]},{"status":"valid","tenderers":[{"id":"0"}]}]},"awards":[{"status":"active","suppliers":[{"id":"0"}]}],"tender":{"status":"cancelled"}}
