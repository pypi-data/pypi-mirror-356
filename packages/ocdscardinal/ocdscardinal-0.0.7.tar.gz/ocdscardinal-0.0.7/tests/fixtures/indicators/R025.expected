{"Tenderer":{"N":{"R025":0.0},"F":{"R025":0.2}}}
